### ISSUE TITLE
Make it easy for analysts to copy domain request data to clipboard

### ISSUE THREAD

**Issue by PaulKuykendall** [cisagov/manage.get.gov#2406]
### Issue description
Right now, Analysts manually copy over domain request information to a Microsoft Planner as part of the review process. This means tediously finding each item listed in the additional context, highlighting the value to copy to clipboard, and then pasting in a template. The template they use is exactly what is in the additional context. This is tedious work that doesn't add value. 

To make analyst lives easier, we want to copy all the relevant data from the domain request into the clipboard matching the format they use. This includes the bolded subtitle names and having everything inline as much as possible to create a condensed view.

### Acceptance criteria

- [ ] There is a "Copy request summary" button added to domain requests in django admin. Button should appear under the "History" button in upper right of page.
- [ ] The button adds the content as described in the below ACs to clipboard
- [ ] Content matches the template in the additional context, the values in <> are meant to provide direction the data that should go in each field
- [ ] One blank line should be added between each head (just like below) and headers should be bold

### Additional context

**Recommendation:** < leave empty >

**Organization Type:** <organization_type>

**Requested Domain:** <requested_domain>

**Current Websites:** 

* <current_website>
* Add Whois links and sources used to confirm that it's an official website for the requesting organization

**Rationale:** < leave empty >

**Alternative Domains:**

* < alternate domain names in a comma separated list>

**Submitter:** 

* <name, title  and contact info>
* Add additional details and links used for verification

**Senior Official:**

* <name, title and contact info in one line>
* Add additional details and links used for verification

**Other Employees:** 

* <contact 1 name, title, contact info>
  * Add additional details and links used for verification
* <contact 2 name, title, contact info>
  * Add additional details and links used for verification

**Comment by abroddrick** [comment]
@Katherine-Osos or @gabydisarli check my first AC  for the name of the button, please feel to provide a better name. Maybe "Copy Planner data"? And let me know where you want the button I am assuming the top left next to the manage domain button

**Comment by vickyszuchin** [comment]
I'm moving this ticket to "Ready" given the story point has been added, and most of us have reviewed it. The scope and ACs are well defined enough. If they need to be tweaked, it should not expand the story point.

**Comment by gabydisarli** [comment]
@Katherine-Osos I'll let you answer, I don't think I know enough about the problem to give a good answer.

**Comment by Katherine-Osos** [comment]
@abroddrick Updated the first AC with more direction.

**Comment by katypies** [comment]
I added a bit of formatting to the template, based on feedback from @amcelya during the sprint about using bullets in particular to make the information easier to read/parse. 

I also removed the descriptions in most of the cases, since we're already getting that information when copying it. But, added bullets to indicate additional information that an Analyst will fill in once they've put this into the Planner card.
